src/foam/lib/PermissionPredicate.js

```diff
@@ -0,0 +1,47 @@
+foam.CLASS({
+  package: 'foam.lib',
+  name: 'PermissionPredicate',
+  extends: 'foam.mlang.predicate.AbstractPredicate',
+  implements: ['foam.core.Serializable'],
+  imports: [
+    'auth',
+  ],
+  properties: [
+    {
+      name: 'args',
+      class: 'StringArray'
+    }
+  ],
+  javaImports: [
+    'foam.nanos.auth.AuthService'
+  ],
+
+  methods: [
+    {
+      name: 'f',
+      javaCode: `
```

Comment:
The PermissionPredicates above, when is x passed in? 
Can we not pass 'x' as the 'obj' in f(obj)?  On the java side will this not be the system context?

Reply:
Fixed